conductor/src/test/java/com/bluelinelabs/conductor/util/TestController.java

```diff
@@ -46,7 +47,7 @@
 
     @NonNull
     @Override
-    protected View onCreateView(@NonNull LayoutInflater inflater, @NonNull ViewGroup container) {
+    protected View onCreateView(@NonNull LayoutInflater inflater, @NonNull ViewGroup container, @Nullable Bundle savedViewState) {
```

Comment:
Why not decouple the logic into two methods, making the primary one as `@CallSuper` required. That could break back-compatibility for a lot of controllers as they select the fork instead.

Reply:
I think it's not a good idea

Reply:
"That could break back-compatibility for a lot of controllers as they select the fork instead."

The main repo also make breaking changes

Reply:
Well fine, lets go with a single method. If somebody decides he need it he will just it to the base class so it wont break fully.